eva-accession-release/src/test/java/uk/ac/ebi/eva/accession/release/io/VariantContextWriterTest.java

```diff
@@ -276,4 +297,63 @@ public void throwIfNonStandardNucleotides() throws Exception {
         File output = temporaryFolder.newFile();
         assertWriteVcf(output, buildVariant(CHR_1, 1000, "C", "U", SNP_SEQUENCE_ONTOLOGY, STUDY_1));
     }
+
+    @Test
+    public void checkDefaultFlags() throws Exception {
+        File output = temporaryFolder.newFile();
+        assertWriteVcf(output,
+                       buildVariant(CHR_1, 1000, "C", "G", SNP_SEQUENCE_ONTOLOGY, false, false, true, true, true,
+                                    STUDY_1));
+
+        String dataLinesRegex = "^[^#]";
+        String dataLinesWithFlagsRegex = dataLinesRegex + ".*(" + VALIDATED_KEY
+                                    + "|" + SUBMITTED_VARIANT_VALIDATED_KEY
+                                    + "|" + ALLELES_MATCH_KEY
+                                    + "|" + ASSEMBLY_MATCH_KEY
+                                    + "|" + SUPPORTED_BY_EVIDENCE_KEY + ")";
+        List<String> dataLines = grepFile(output, dataLinesWithFlagsRegex);
+
+        assertEquals(0, dataLines.size());
+    }
+
+    @Test
+    public void checkNonDefaultValidatedFlag() throws Exception {
+        assertFlagIsPresent(VALIDATED_KEY);
+    }
+
+    private void assertFlagIsPresent(String flagRegex) throws Exception {
+        File output = temporaryFolder.newFile();
+        assertWriteVcf(output,
+                       buildVariant(CHR_1, 1000, "C", "G", SNP_SEQUENCE_ONTOLOGY, true, true, false, false, false,
+                                    STUDY_1));
+
+        String dataLinesRegex = "^[^#]";
+        String dataLinesWithValidatedRegex = dataLinesRegex + somewhereSurroundedByTabOrSemicolon(flagRegex);
+        List<String> dataLines;
+        dataLines = grepFile(output, dataLinesWithValidatedRegex);
+        assertEquals(1, dataLines.size());
+    }
+
+    private String somewhereSurroundedByTabOrSemicolon(String regex) {
+        return ".*[\t;]" + regex + "([\t;].*|$)";
```

Comment:
Shouldn't it be a double backslash followed by t for the tab character class?

Reply:
It doesn't matter, I added a test to show that